primitives/manta/Cargo.toml

```diff
@@ -16,9 +16,6 @@ log = "0.4.16"
 scale-info = { version = "2.1.2", default-features = false, features = ["derive"] }
 smallvec = "1.8.0"
 
-# manta-rs dependencies
-manta-accounting = { git = "https://github.com/manta-network/manta-rs.git", tag = "v0.5.4", default-features = false }
-
 # Substrate primitives
 frame-benchmarking = { git = 'https://github.com/paritytech/substrate.git', branch = "polkadot-v0.9.26", default-features = false, optional = true }
```

Comment:
`'`

Reply:
Seems this pr is going to replace all `'` to `"`

Reply:
That's not the intent of this PR. All of these type of changes are either from taplo fmt or accidental.
We can do this separately if we want to.